src/js/components/landing/blocks/LandingRequirementsBody.jsx

```diff
@@ -133,73 +108,21 @@ export default class LandingRequirementsBody extends React.Component {
                         File F: Sub-award Attributes data
                     </li>
                 </ul>
+                <p>Submission information is available on the DAIMS page of the Data Transparency site for the Bureau of the Fiscal Service. </p>
```

Comment:
According to the document attached to the story, `DAIMS` should be a link to https://fiscal.treasury.gov/fsservices/gov/data-trans/dt-daims.htm